src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -315,6 +315,13 @@ export default class TableComponent extends ComponentBase implements IComponent,
 		this._table.autosizeColumns();
 	}
 
+	public focus(): void {
+		if (!this._table.grid.getActiveCell()) {
+			this._table.grid.setActiveCell(0, 0);
+		}
+		this._table.grid.getActiveCellNode().focus();
```

Comment:
What happens if the table is empty?

Reply:
There was an error that it can't call focus() on null. Added a check for empty table :)